core/src/main/resources/error/error-classes.json

```diff
@@ -536,7 +536,7 @@
     ],
     "sqlState" : "23505"
   },
-  "ELEMENT_AT_BY_INDEX_ZERO" : {
+  "INVALID_INDEX_OF_ZERO" : {
```

Comment:
Highly likely, it should be moved since error classes must be sorted in the file.

Reply:
got it, will fix it soon